sql/core/src/main/scala/org/apache/spark/sql/execution/command/cache.scala

```diff
@@ -21,7 +21,6 @@ import java.util.Locale
 
 import org.apache.spark.sql.{Dataset, Row, SparkSession}
 import org.apache.spark.sql.catalyst.TableIdentifier
-import org.apache.spark.sql.catalyst.analysis.NoSuchTableException
```

Comment:
Shall we revert this change in this file?

Reply:
Sure.